objects/Master.py

```diff
@@ -0,0 +1,66 @@
+import sys
+import shutil
+import os
+import asyncio
+from hyperopt import fmin, tpe
+from hyperopt.mongoexp import MongoTrials
+from .OPTSubprocess import OPTSubprocess
+# import from directory up
+sys.path.append('..')
+from NWT_SUBMIT.NWTOPT_FILES.objective import objective
+from config.HParams import MF5HParams
+
+class Master(OPTSubprocess):
+    def __init__(self, id, logger, cwd, ip, port, key, random, trials):
+        super().__init__('master', id, logger)
+        self.cwd = cwd
+        try:
```

Comment:
you can also do like this to avoid the `try/except`

```python
if os.path.exists(somedir): 
    shutil.rmtree(somedir) 
os.mkdir(somedir)

I don't like relying on error trapping for more known behavior